src/three-components/SmoothControls.ts

```diff
@@ -304,6 +317,16 @@ export class SmoothControls extends EventDispatcher {
     this[$spherical].copy(this[$targetSpherical]);
   }
 
+  /**
+   * Resets the input sensitivity based on model scale, here represented by
```

Comment:
Does it only affect input? It seems like `$zoomMeters` is also used when `update` is invoked (which is usually caused by configuration changes)

Reply:
Yes, it's used as a normalization for radius to affect acceleration of the smooth controls (which I would still consider a type of input sensitivity). I can't really think of a better way to explain it.

Reply:
For me, the word "input" suggests "user input," or something the person clicking / tapping / interacting with the page is doing. In practice, `$zoomMeters` is universally applied regardless of the origin of the change. "Zoom" might be a fine substitute that avoids the ambiguity.